java/src/jmri/jmrix/lenz/XNetReply.java

```diff
@@ -715,10 +715,6 @@ public boolean onTurnoutFeedback(int accessoryNumber, Function<FeedbackItem, Boo
             return Optional.empty();
         }
         FeedbackItem item = new FeedbackItem(this, accessoryNumber, r);
-        if (item.getAccessoryStatus() > 2) {
```

Comment:
Thanks for the correction.

Reply:
Still a question: is this combination sent while the turnout w/feedback is still moving - none of the end switches report turnout's position ?

Reply:
For turnouts with feedback, If the turnout is reported as still moving, the motion complete bit will be 0.  Turnouts without feedback always set the motion complete bit to 1.

The invalid combination may never actually be sent, but it is defined in the XpressNet manual as being "invalid".  Operationally, XNetTurnout objects treat both the 00 case and 11 case the same, but the XpressNet monitor treats them differently.  I caught this while refactoring the Monitor code to use the Feedback items.